extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -18,6 +18,8 @@ const BackbeatClient = require('../../../lib/clients/BackbeatClient');
 const QueueEntry = require('../utils/QueueEntry');
 const CredentialsManager = require('../../../credentials/CredentialsManager');
 
+const LIMIT = 10;
```

Comment:
I think this translates to rate limiting for us. Our rate limiting api will/can probably tune this, we can make it configurable then.